Makefile

```diff
@@ -110,7 +111,7 @@ test: fmt vet lint ## Run tests.
 build: craned crane-agent metric-adapter
 
 .PHONY: all
-all: generate test craned  crane-agent metric-adapter
+all: generate test craned  crane-agent metric-adapter image-dashboard
```

Comment:
shouldn't be dashboard like others?
make all are not building images.

Reply:
My fault, i put it into images.